package/yast2-packager.spec

```diff
@@ -74,12 +74,10 @@ Requires:	libyui_pkg
 # ensure that 'checkmedia' is on the medium
 Recommends:	checkmedia
 
-Provides:	yast2-config-package-manager
-Obsoletes:	yast2-config-package-manager
-Provides:	y2t_spkg y2t_inst-packages y2pkginf y2c_spkg
-Obsoletes:	y2t_spkg y2t_inst-packages y2pkginf y2c_spkg
-Provides:	yast2-trans-package-manager yast2-trans-inst-packages
-Obsoletes:	yast2-trans-package-manager yast2-trans-inst-packages
+# for registering media add-ons
+# (soft dependency because openSUSE does not contain the registration module)
+Recommends:     yast2-registration
```

Comment:
Couldn't this cause any unwanted warning in openSUSE in some situations? What about using conditionals in the spec file? Just asking, I actually don't know if it's a better solution, I have no clue about packaging.